crates/context_aware_config/src/helpers.rs

```diff
@@ -310,6 +321,49 @@ pub fn add_config_version(
     Ok(version_id)
 }
 
+#[cfg(feature = "high-performance-mode")]
+pub async fn put_config_in_redis(
+    version_id: i64,
+    state: Data<AppState>,
+    tenant: Tenant,
+    db_conn: &mut PooledConnection<ConnectionManager<PgConnection>>,
+) -> superposition::Result<()> {
+    let config = generate_cac(db_conn)?;
+    let redis_config = serde_json::to_string(&json!(config)).map_err(|e| {
+        log::error!("failed to convert cac config to string: {}", e);
+        unexpected_error!("could not convert cac config to string")
+    })?;
+    let config_key = format!("{}::cac_config", *tenant);
+    let max_created_key = format!("{}::cac_config::max_created_at", *tenant);
```

Comment:
Can we change the keyName, looks misleading.

Reply:
How so?

Reply:
What is the meaning of this max_created_key?

Reply:
This stores the header value for max_created_at